crypto-ballerina/Package.md

```diff
@@ -1,5 +1,17 @@
 ## Package Overview
 
-This package provides the necessary utilities that are required to hash content using different hashing mechanisms and algorithms. 
+The Crypto library is one of the standard library modules of the [Ballerina](https://ballerina.io/) language.
 
-For information on the operations, which you can perform with this package, see the below **Functions**. For an example on the usage of the operations, see the [Cryptographic Operations Example](https://ballerina.io/learn/by-example/crypto.html).
+The Ballerina Crypto module facilitates APIs to do operations like hashing, HMAC generation, checksum generation, encryption, decryption, digitally signing data and verifying digitally signed data etc. with different cryptographic algorithms.
+
+For more information go to [The Crypto Module](https://docs.central.ballerina.io/ballerina/crypto/latest).
```

Comment:
Shall we remove the API doc link from Package.MD

Reply:
Fixed with https://github.com/ballerina-platform/module-ballerina-crypto/pull/172/commits/0c61412e2f212771b1e88a6271356c7d89ff2d03